google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, global_configs):
             api_name=self.API_NAME)
 
         self.rate_limiter = RateLimiter(
-            self.global_configs.get('max_admin_api_calls_per_day'),
+            self.global_configs.get('max_admin_api_calls_per_100_seconds'),
```

Comment:
I can't find carise's change management doc yet, but would a check for forseti version number help to manage this change here?
```
   if version <= 1.1.0:
       then exit
```